apps/ledger-live-mobile/src/navigation/DeeplinksProvider.tsx

```diff
@@ -447,7 +452,7 @@ export const DeeplinksProvider = ({
         getStateFromPath: (path, config) => {
           const url = new URL(`ledgerlive://${path}`);
           const { hostname, pathname } = url;
-          const platform = pathname.split("/")[1];
+          const [platform] = pathname.split("/");
```

Comment:
It would be good check this with @jules-grenier-ledger as we are moving from 0 element to 1. Not sure if we are broken their logic.

Reply:
Ah yes, will revert this, thank you!